client/state/data-layer/wpcom/read/lists/items/index.js

```diff
@@ -23,7 +27,7 @@ registerHandlers( 'state/data-layer/wpcom/read/lists/items/index.js', {
 				),
 			onSuccess: ( action, apiResponse ) =>
 				receiveReaderListItems( apiResponse.list_ID, apiResponse.items ),
-			onError: ( action, error ) => errorNotice( String( error ) ),
+			onError: () => noop,
```

Comment:
Doing a `noop` because the linter complains when `onError` is removed altogether.